src/main/java/io/getunleash/util/UnleashConfig.java

```diff
@@ -716,7 +736,7 @@ public String getDefaultSdkVersion() {
             String version =
                     Optional.ofNullable(getClass().getPackage().getImplementationVersion())
                             .orElse("development");
-            return "unleash-client-java:" + version;
```

Comment:
I don't think we're using it anywhere so should be safe to rename it to a common convention with other SDKs. The convention is inspired by npm versions since Node is a dominant part of our stack.

Reply:
Yeah I also think that's fine